web/src/main/java/org/apache/shiro/web/util/WebUtils.java

```diff
@@ -94,7 +95,7 @@ public class WebUtils {
      * @see javax.servlet.ServletRequest#getCharacterEncoding
      */
     public static final String DEFAULT_CHARACTER_ENCODING = "ISO-8859-1";
-
+    
```

Comment:
I know it's a nit, but blank space here should be removed

Reply:
Blank space has been removed